### ISSUE TITLE
Error: Unsupported argument on _enventhub.tf , in resource "azurerm_eventhub_namespace"

### ISSUE THREAD

**Issue by rishab-sgid** [hashicorp/terraform-provider-azurerm#18908]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Community Note

* Please vote on this issue by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

1.3.3

### AzureRM Provider Version

3.8.0

### Affected Resource(s)/Data Source(s)

azurerm_eventhub_namespace

### Terraform Configuration Files

```hcl
resource "azurerm_eventhub_namespace" "eventhub_namespace" {
 
    name                = "Eventhub"
    location            = "Location"
    resource_group_name = "MyRGName"
    sku                 = "Standard"
    capacity            = "1"

    network_rulesets {
      default_action                 = "Deny"
      trusted_service_access_enabled = true
      # virtual_network_rule {
      #   subnet_id = azurerm_subnet.subnet[each.value.subnet_id_name].id
      # }
    }

    minimum_tls_version           = "1.2"
    public_network_access_enabled = false
 
}
```

### Debug Output/Panic Output

```shell
An argument named "public_network_access_enabled" is not expected here.
##[error]Terraform command 'validate' failed with exit code '1'.:  Unsupported argument |  Unsupported argument
##[error]
Error: Unsupported argument

  on _enventhub.tf line 106, in resource "azurerm_eventhub_namespace" "eventhub_namespace":
 106:     minimum_tls_version           = "1.2"

An argument named "minimum_tls_version" is not expected here.


Error: Unsupported argument

  on _enventhub.tf line 107, in resource "azurerm_eventhub_namespace" "eventhub_namespace":
 107:     public_network_access_enabled = "false"

An argument named "public_network_access_enabled" is not expected here.
```

### Expected Behaviour

"public_network_access_enabled" , "minimum_tls_version" should be applied to eventhub resource in Azure Portal and they are part of attribute list in terraform eventhub resource (https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/eventhub_namespace)

### Steps to Reproduce

terraform init
terraform validate

### References

NA

**Comment by jayendranarumugam** [comment]
Both `public_network_access_enabled` and `minimum_tls_version` are supported from [version 3.17.0](https://github.com/hashicorp/terraform-provider-azurerm/blob/v3.17.0/CHANGELOG.md)

**Comment by xiaxyi** [comment]
Thanks @jayendranarumugam , The property was introduced to Terraform provider 3.23.0, it's only available after version 3.23.0.

you can refer to this pr for more information:
https://github.com/hashicorp/terraform-provider-azurerm/pull/18314

This is the change log:https://github.com/hashicorp/terraform-provider-azurerm/blob/v3.23.0/CHANGELOG.md

**Comment by rishab-sgid** [comment]
I confirm the issue is resolved after upgrading the provider azurerm version to 3.23.0 and higher.
